config.toml

```diff
@@ -4,7 +4,7 @@ title = "CDEvents"
 enableRobotsTXT = true
 
 # Hugo allows theme composition (and inheritance). The precedence is from left to right.
-theme = ["github.com/google/docsy", "github.com/google/docsy/dependencies"]
+theme = ["hugo-redirect", "github.com/google/docsy", "github.com/google/docsy/dependencies"]
```

Comment:
You can use "github.com/gcc42/hugo-redirect" instead, and drop the `.gitmodules` file completely.
After changing the `config.toml`, you can run `hugo mod tidy` and it will fetch the module for you.

Reply:
Looks like that has resolved the clash between the old and new versions of content/en/docs, unless it’s a race condition.